communities.json

```diff
@@ -771,6 +771,13 @@
       "quote": "Discord has been amazing for accelerating the development of React Spring and React Three Fiber. Screen sharing and instant communication have been pivotal in resolving bugs and coordinating with our contributors.",
       "inviteCode": "poimandres",
       "githubUrl": "https://github.com/pmndrs"
+    },
+    {
+      "logo": "radix.svg",
+      "title": "Radix",
+      "quote": "We use Discord to connect with our community and provide support. Many of our accessible components have been tested and improved by this close collaboration with our users. Thank you!",
+      "inviteCode": "7Xb99uG",
```

Comment:
Did you mean for this to go to the "Modulz" server?

Reply:
Hey @jamiebuilds I left a reply on my [other PR](https://github.com/discord/discord-open-source/pull/319#discussion_r573186507), but will copy it here too:
The way we're using Discord at Modulz is that we have one server and different categories for our sub-products (Stitches and Radix). 

The invite I added here is specifically to the welcome channel Radux category.

Is that an issue? I was thinking about creating different servers for different sub-products, but thought this would be easier to manage.

Reply:
That's fine, can you double check that this invite code does not have an expiration date tied to it?

Reply:
@jamiebuilds checked, no expiration date 👍 Thanks!

![image](https://user-images.githubusercontent.com/372831/107429097-d5cde500-6b23-11eb-9ce0-54232ccec711.png)